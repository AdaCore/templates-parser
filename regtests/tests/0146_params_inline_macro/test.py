from test_support import *

run('testme', ["testme146.tmplt"])
