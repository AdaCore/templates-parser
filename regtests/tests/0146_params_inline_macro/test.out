
Test INLINE/EXTENDS into a MACRO


1/3:
L1: $1:ONE $2: $3:THREE P1:ONE P2: P3:THREE
L2: $1:ONE $2: $3:THREE P1:ONE P2: P3:THREE L4: $1:ONE $2: $3:THREE P1:ONE P2: P3:THREE L5: $1:ONE $2: $3:THREE P1:ONE P2: P3:THREE L7: $1:ONE $2: $3:THREE P1:ONE P2: P3:THREE
L8: $1:ONE $2: $3:THREE P1:ONE P2: P3:THREE

LI: $1:ONE $2: $3:THREE $4:azerty P1: P2: P3:
L9: $1:ONE $2: $3:THREE $4: P1:ONE P2: P3:THREE


2:
L1: $1: $2:TWO $3: P1: P2:TWO P3:
L2: $1: $2:TWO $3: P1: P2:TWO P3: L4: $1: $2:TWO $3: P1: P2:TWO P3: L5: $1: $2:TWO $3: P1: P2:TWO P3: L7: $1: $2:TWO $3: P1: P2:TWO P3:
L8: $1: $2:TWO $3: P1: P2:TWO P3:

LI: $1: $2:TWO $3: $4:azerty P1: P2: P3:
L9: $1: $2:TWO $3: $4: P1: P2:TWO P3:

